BookApi/BookApi/Repository/BookRepository.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using BookApi.Models;
+using Microsoft.EntityFrameworkCore;
```

Comment:
```suggestion
using BookApi.Models;
using Microsoft.EntityFrameworkCore;
```
VSCode shows these as unused imports :)